src/js/server/controllers/apply/concerns/appliable-concern.ts

```diff
@@ -0,0 +1,32 @@
+import { HackerApplicationInstance } from 'js/server/models';
```

Comment:
Already reviewed in previous pull

Reply:
I've merged in the changes from the other branch